workspace/test.ml

```diff
@@ -3,24 +3,51 @@ open OUnit
 let _esc_lf s =
   s |> Str.global_replace (Str.regexp_string "\n") "<:LF:>";;
 
-(* TODO Fix missing `<COMPLETEDIN::>` *)
+let cw_print_success = function
+  | _ -> print_endline ("\n<PASSED::>Test passed")
```

Comment:
`function` should only be used when there is a pattern-matching for the last argument. This function (and many other functions) should be written as
```ocaml
let cw_print_success () = print_endline "\n<PASSED::>Test passed"
```
Unnecessary parentheses should also be removed.